fastapi_contrib/db/client.py

```diff
@@ -31,6 +31,15 @@ def get_collection(self, collection_name: str) -> Collection:
         collection = self.get_collection(collection_name)
         return await collection.insert_one(data, session=session)
 
+    async def update(
+        self, model: MongoDBModel, session=None, include=None, exclude=None
+    ) -> InsertOneResult:
+        data = model.dict(include=include, exclude=exclude)
+        doc_id = data.pop("id")
```

Comment:
I might want to search for the record I chose to update not only by id of the record, but use multiple conditions to filter out one needed record. In our case (at least for now) it doesn't matter whether we update one record or multiple records at once -> we should be able to do both. 

Check out `get` function which uses `**kwargs` to store all the filtering info and please refactor this to be similar.